README.md

```diff
@@ -1,6 +1,6 @@
 # nvidia-snatcher [![ci](https://github.com/jef/nvidia-snatcher/workflows/ci/badge.svg)](https://github.com/jef/nvidia-snatcher/actions?query=workflow%3Aci) [![discord](https://img.shields.io/discord/756303724095471617.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/Cyc7nrz)
 
-[FAQ](#FAQ) | [Issues](https://github.com/jef/nvidia-snatcher/issues) | [Wiki](https://github.com/jef/nvidia-snatcher/wiki)
+[FAQ](#FAQ) | [Issues](https://github.com/jef/nvidia-snatcher/issues) | [Wiki](https://github.com/jef/nvidia-snatcher/wiki) | [Discord](https://discord.gg/Cyc7nrz)
```

Comment:
```suggestion
# nvidia-snatcher [![ci](https://github.com/jef/nvidia-snatcher/workflows/ci/badge.svg)](https://github.com/jef/nvidia-snatcher/actions?query=workflow%3Aci) [![discord](https://img.shields.io/discord/756303724095471617.svg?label=join&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/Cyc7nrz)

[FAQ](#FAQ) | [Issues](https://github.com/jef/nvidia-snatcher/issues) | [Wiki](https://github.com/jef/nvidia-snatcher/wiki)	[FAQ](#FAQ) | [Issues](https://github.com/jef/nvidia-snatcher/issues) | [Wiki](https://github.com/jef/nvidia-snatcher/wiki)
```

Makes something like this: [![discord](https://img.shields.io/discord/756303724095471617.svg?label=join&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/Cyc7nrz)

I don't know the best verbiage, but maybe you can figure that out

Reply:
Sure yep that works! TBH I just found it a little hidden / hard to discover when the badge inline with the heading of the readme. 

Would you be comfortable with addding the change to the badge as above & then a section in the FAQ along the lines of `if you need additional help in getting set up, pop into our discord...` etc...?

Reply:
> TBH I just found it a little hidden / hard to discover when the badge inline with the heading of the readme.

I think this good feedback! I really don't know what a user looks for 😆 Just that I thought it looked slick.

I think adding to the FAQ is a good idea! I also feel like no matter how many time we will post help, someone will miss it. But this should be plenty enough.